src/phoenix/experimental/evals/templates/template.py

```diff
@@ -122,7 +127,8 @@ def map_template(dataframe: pd.DataFrame, template: PromptTemplate) -> "pd.Serie
     try:
         prompts = dataframe.apply(
             lambda row: template.format(
-                variable_values={var_name: row[var_name] for var_name in template.variables}
+                variable_values={var_name: row[var_name] for var_name in template.variables},
+                **options,
```

Comment:
I see, I get where this is going - but I do think we can still strongly type this